### DISCUSSION TITLE
Super User Permission

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#5841] [Q&A]
How to grant user access for users, groups and permissions without the Super User checkbox, how to hide the Super User action please, any way to do that ?
![Capture](https://user-images.githubusercontent.com/29832976/163487679-a2b31a73-383c-473e-9c4c-ec44808868c0.PNG)

**Comment by edalzell** [comment]
Why do you need to do that?

**Reply by HazemMohamed252** [reply]
Because i want to create persmission for example : Admin and grant him all permissions including the users, groups and permissions module but prevent him from take action on super user and see all the basic modules like collections, blueprint ans so on ...
I need a way and i try to access the core folder for statamic to do that but i can not and not reached for it. Any way please ??
